packaging/rpm/make-microshift-images-rpm.sh

```diff
@@ -23,26 +26,33 @@ ARCHITECTURES=${ARCHITECTURES:-"x86_64 arm64 arm ppc64le riscv64"}
 build() {
   cat >"${RPMBUILD_DIR}"SPECS/microshift-images.spec <<EOF
 %global baseVersion ${BASE_VERSION}
-
+%global github_tag ${GITHUB_TAG}
 EOF
```

Comment:
WIP: this all needs to be changed into generating a .yaml file, and calling paack.py